packages/perseus/src/widgets/interactive-graphs/mafs-graph.tsx

```diff
@@ -161,40 +165,69 @@ export const MafsGraph = (props: MafsGraphProps) => {
                         </Mafs>
                     </View>
                 </View>
-                {renderGraphControls({state, dispatch})}
+                {renderGraphControls({state, dispatch, width})}
             </View>
         </GraphConfigContext.Provider>
     );
 };
 
 const renderPointGraphControls = (props: {
-    state: InteractiveGraphState;
+    state: PointGraphState;
     dispatch: (action: InteractiveGraphAction) => unknown;
+    width: number;
 }) => (
-    <Button
-        kind="secondary"
+    <View
         style={{
-            width: "100%",
-            marginLeft: "20px",
-        }}
-        onClick={() => {
-            props.dispatch(actions.pointGraph.addPoint([0, 0]));
+            flexDirection: "row",
+            width: props.width,
```

Comment:
Why do we need to specify the `width` now? It seems like `width: 100%` should work, given that this container is `display: flex`.

Reply:
Width: 100% takes up the whole container width and spills outside the graph 😱. This seems to make things show up correctly